### ISSUE TITLE
Move photo gallery to its own page

### ISSUE THREAD

**Issue by iteles** [mosteirodelandim/landim-website#17]
This will require more thought in terms of organisation of photos and is low priority for now.

**Comment by LuchoTurtle** [comment]
@iteles We need a layout plan for this one. Are the pictures to be divided? I'd need a simple draft on how the layout of this page would be and I could work on it. The only thing that is keeping me from progressing is whether you want to divide into spaces or make a simple Image Carousel with everything in it (I don't think it's the best option). Something like the image in issue #2 would suffice :)

**Comment by iteles** [comment]
Ok, leave this with me! I’ll do a little research for #19 and that should provide some inspiration. 

I’m torn because on the one hand, a walk-through of everything by scrolling through an image gallery is perfect for people looking for an event venue for example and on the other, there will be people who only care about the space itself (without event photos) or the gardens. But you can’t be everything to everything to everyone so I’ll have a think about this 👍🏻

**Comment by LuchoTurtle** [comment]
@iteles about #19, it already has been implemented. I've provided a layout with _lorem impsum_. I tried to maintain the same style and layout consistent with the rest of the site. What's really missing is the content of the page (same goes for Gardens and Wine). To be honest, I can do the layout for the Photogallery but, before deployment, we just need to focus on finishing up on two things:

- Written content of each page
- Choosing the images across the site (they are not compressed nor optimized so far for obvious reasons)

**Comment by iteles** [comment]
@LuchoTurtle As referred to in #19, the research is on competitor websites in order to aid with content creation, which should also inform the decision here.

As for what needs to be done pre-deployment, we're kind of confusing things by adding random things to a comment in an un-related issue without a reference, that's why we have a [deployment milestone](https://github.com/mosteirodelandim/landim-website/milestone/1) - please do let me know if there's anything in there that you _don't_ think needs to be done before deployment.

On _this_ issue, I agree with you, it _doesn't_ need to be done prior to deployment, that's why I didn't put it in the milestone 👍
